service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerAboutToSubmitTest.java

```diff
@@ -117,7 +117,7 @@ void shouldRemoveTemporaryFieldsWhenPresent() {
             .build());
 
         assertThat(callbackResponse.getData())
-            .doesNotContainKey("amountToPay")
+            .containsEntry("amountToPay", "233300")
```

Comment:
This entire test seems pointless now, it is not checking that anything has been removed.

Reply:
changed test name to be more meaningful. Let me know what you think!

Reply:
maybe something more like should retain payment information as we are not just keeping amountToPay

Reply:
done